### PR TITLE
Add deploy an existing frontend application page (with Next.js section)

### PR DESCRIPTION
**Pull request by jennifertrin**
- Add deploy an existing frontend application page
- Add a section on how to deploy an existing Next.js application 

Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [X] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [X] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: docs/developer-docs/frontend/existing-frontend.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+`npx run build`
+
+You should reference your `package.json` scripts section to find the correct build command. 
+
+This should now generate an `out` folder which consists of the static assets that make up the website.
+
+In the next step, we will instruct the Internet Computer to deploy the website on-chain using these static files. 
+
+:::info
+When deploying on the Internet Computer, these static files are not public to anyone including the nodes. Only the WASM file which is a binary instruction file which does not leak any of your code is public to nodes.
```
Comment (jessiemongeon1) [inline docs/developer-docs/frontend/existing-frontend.md:61]:
```suggestion
When deploying on ICP, these static files are not public to anyone including the nodes. Only the Wasm file which is a binary instruction file which does not leak any of your code is public to nodes. 
```

File: docs/developer-docs/frontend/existing-frontend.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        },
+        "source": ["out"],
+        "type": "assets"
+      },
+      "output_env_file": ".env"
+    }
+}
+```
+
+`dfx.json` is the configuration file for deploying all of your code to canister smart contracts on the Internet Computer.
```
Comment (jessiemongeon1) [inline docs/developer-docs/frontend/existing-frontend.md:83]:
```suggestion
`dfx.json` is the configuration file for deploying all of your code to canister smart contracts on ICP.
```

File: docs/developer-docs/frontend/existing-frontend.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+Please note that you can adjust the following:
+
+`app` - name of the canister smart contract
```
Comment (jessiemongeon1) [inline docs/developer-docs/frontend/existing-frontend.md:87]:
```suggestion
`app`: Refers to the name of the canister smart contract.
```

File: docs/developer-docs/frontend/existing-frontend.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+and folder:
+
+```
+ "source": ["out"],
+```
+
+### Step 3: Generate Candid definitions 
+
+Run the following command to generate the correct types.
```
Comment (jessiemongeon1) [inline docs/developer-docs/frontend/existing-frontend.md:103]:
```suggestion
Run the following command to generate the correct types:
```

**Review by jessiemongeon1 — APPROVED** [review]

**Comment by jennifertrin** [comment]
@jessiemongeon1 Thank you, Jesse for the help and pointing out correct phrase formatting!

**Pull request merged**